pkg/scheduler/algorithm/predicates/predicates_test.go

```diff
@@ -1387,11 +1388,66 @@ func TestPodFitsSelector(t *testing.T) {
 			fits: false,
 			test: "Pod with an invalid value in Affinity term won't be scheduled onto the node",
 		},
+		{
```

Comment:
Could you please also add a test case with multiple selector terms? For example a case where there are multiple `MatchExpressions` and `MatchFields` terms?

Reply:
done :)